### ISSUE TITLE
setup.py fails to run with Python 3 and a non-unicode locale

### ISSUE THREAD

**Issue by ghisvail** [nschloe/meshio#112]
The Debian builders do not have a unicode locale set, and therefore your `setup.py` fails to run with the system `python3` interpreter:
```
>>> python3 setup.py clean
running clean
```
```
>>> LC_ALL=C python3 setup.py clean
Traceback (most recent call last):
  File "setup.py", line 11, in <module>
    exec(f.read(), about)
  File "/usr/lib/python3.5/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 73: ordinal not in range(128)
```

It works fine with Python 2.

**Comment by nschloe** [comment]
Sounds like I forgot to add the utf-8 header
```python
# -*- coding: utf-8 -*-
```
somewhere. I'll have a look...

**Comment by nschloe** [comment]
Should be fixed now (1.8.13).

I've also added some minor things that will hopefully make Debian packaging easier.
